setup.cfg

```diff
@@ -18,11 +18,11 @@ precision = 2
 
 [isort]
 not_skip = __init__.py
-known_third_party = appdirs, backports, faker,factory, faker, freezegun, future, hamster_lib, icalendar, past, pytest, pytest_factoryboy,
+known_third_party = appdirs, configparser, faker,factory, faker, freezegun, future, hamster_lib, icalendar, past, pytest, pytest_factoryboy,
```

Comment:
well.. its a nother py2 vs3 case :( under py2 it is a 3rd party lib, under python3 it is not
so there is no "right way" to do this i guess :/